webshooter.py

```diff
@@ -117,7 +119,7 @@ def jekyll_build(path):
     print(options_prompt)
     while static not in {"1", "2"}:
```

Comment:
Can you convert `{"1", "2"}` into `("1", "2")` for me? I find this syntax... concerning. I think it's making a set literal? The python idiom for this is to use a `tuple`, not a `set`.